pkg/db/vulnerability_metadata.go

```diff
@@ -1,20 +1,30 @@
 package db
 
+// VendorMetadata captures non-standard CVSS fields that vendors can sometimes
+// include when providing CVSS information.  This vendor-specific metadata type
+// allows to capture that data for persisting into the database
+type VendorMetadata interface{}
+
 // VulnerabilityMetadata represents all vulnerability data that is not necessary to perform package-to-vulnerability matching.
 type VulnerabilityMetadata struct {
 	ID           string   // The identifier of the vulnerability or advisory
 	RecordSource string   // The source of the vulnerability information
 	Severity     string   // How severe the vulnerability is (valid values are defined by upstream sources currently)
 	Links        []string // URLs to get more information about the vulnerability or advisory
 	Description  string   // Description of the vulnerability
-	CvssV2       *Cvss    // Common Vulnerability Scoring System V2 values
-	CvssV3       *Cvss    // Common Vulnerability Scoring System V3 values
+	Cvss         []Cvss   // Common Vulnerability Scoring System values
 }
 
 // Cvss contains select Common Vulnerability Scoring System fields for a vulnerability.
 type Cvss struct {
+	VendorMetadata VendorMetadata
```

Comment:
The CVSS information may not be coming specifically from a vendor, so `VendorMetadata` could be inaccurate in some cases. Maybe rename to `Metadata`?

Reply:
I thought the idea was to add vendor-specific metadata here. Buy you are saying that other metadata might not come from a vendor and I'm not sure I've seen an example of that. Do you have an example of a feed that provides metadata that is _not_ a vendor?

Reply:
I don't have any handy, but we do have providers of CVSS info that don't come from vendors (NVD and VulnDB) --naming this field "vendor..." would somewhat preclude adding extra data from those sources here.

Reply:
That's right, NVD and VulnDB do provide CVSS. In the case of VulnDB, it does provide "vendor metadata" (explicitly labeling it as such) and this is already captured and handled as "vendor metadata" in the accompanying PR for grype-db-builder. 
And NVD does not provide vendor (or custom) fields for CVSS. At least, I haven't spotted any custom field that we un-marshal that needs to be captured. I've also double checked that is the case from the feed JSON data coming in. 

If you feel strongly about naming this field `"Metadata"` instead of `"VendorMetadata"` that is fine with me - I just want to clarify that I haven't been able to find examples of CVSS information that is not coming from a vendor.

Do you still want me to rename this?

Reply:
from an offline conversation + this above ^, let's leave this as vendor metadata 👍